### PR TITLE
added proof(correctness)

### PR DESCRIPTION
**Pull request by khushal87**

### CODE REVIEW

File: Euclidean_Algorithm/README.md [hunk 1]
```diff
@@ -22,7 +22,37 @@ The Euclidean algorithm is based on the principle that the greatest common divis
 Euclid's method for finding the greatest common divisor (GCD) of two starting lengths BA and DC, both defined to be multiples of a common "unit" length. The length DC being shorter, it is used to "measure" BA, but only once because remainder EA is less than DC. EA now measures (twice) the shorter length DC, with remainder FC shorter than EA. Then FC measures (three times) length EA. Because there is no remainder, the process ends with FC being the GCD. On the right Nicomachus' example with numbers 49 and 21 resulting in their GCD of 7
 
 
+## Correctness-Proof
+First, notice that in each iteration of the Euclidean algorithm the second argument strictly decreases, therefore (since the arguments are always non-negative) the algorithm will always terminate.
 
+For the proof of correctness, we need to show that gcd(a,b)=gcd(b,a mod b)
+for all a≥0, b>0
+
+.
```
Comment (jainaman224) [inline Euclidean_Algorithm/README.md]: What are these in new lines.

Comment (khushal87) [reply Euclidean_Algorithm/README.md]: They are just black spaces

Comment (jainaman224) [reply Euclidean_Algorithm/README.md]: Please remove them.

Comment (khushal87) [reply Euclidean_Algorithm/README.md]: Done

**Pull request merged**